plugins/connection/pct_remote.py

```diff
@@ -0,0 +1,396 @@
+        env: [{name: ANSIBLE_PARAMIKO_HOST_KEY_AUTO_ADD}]
+        ini:
+          - {key: host_key_auto_add, section: paramiko_connection}
+        type: boolean
+      look_for_keys:
+        default: True
+        description: 'False to disable searching for private key files in ~/.ssh/'
+        env: [{name: ANSIBLE_PARAMIKO_LOOK_FOR_KEYS}]
+        ini:
+        - {key: look_for_keys, section: paramiko_connection}
+        type: boolean
+      proxy_command:
+        default: ''
+        description:
+            - Proxy information for running the connection via a jumphost
+            - Also this plugin will scan 'ssh_args', 'ssh_extra_args' and 'ssh_common_args' from the 'ssh' plugin settings for proxy information if set.
+        type: string
+        env: [{name: ANSIBLE_PARAMIKO_PROXY_COMMAND}]
+        ini:
+          - {key: proxy_command, section: paramiko_connection}
+        vars:
+          - name: ansible_paramiko_proxy_command
+      ssh_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        default: ''
+        type: string
+        ini:
+            - section: 'ssh_connection'
+              key: 'ssh_args'
+        env:
+            - name: ANSIBLE_SSH_ARGS
+        vars:
+            - name: ansible_ssh_args
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      ssh_common_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        type: string
+        ini:
+            - section: 'ssh_connection'
+              key: 'ssh_common_args'
+        env:
+            - name: ANSIBLE_SSH_COMMON_ARGS
+        vars:
+            - name: ansible_ssh_common_args
+        cli:
+            - name: ssh_common_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      ssh_extra_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        type: string
+        vars:
+            - name: ansible_ssh_extra_args
+        env:
+          - name: ANSIBLE_SSH_EXTRA_ARGS
+        ini:
+          - key: ssh_extra_args
+            section: ssh_connection
+        cli:
+          - name: ssh_extra_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      pty:
+        default: True
+        description: 'SUDO usually requires a PTY, True to give a PTY and False to not give a PTY.'
+        env:
+          - name: ANSIBLE_PARAMIKO_PTY
+        ini:
+          - section: paramiko_connection
+            key: pty
+        type: boolean
+      record_host_keys:
+        default: True
+        description: 'Save the host keys to a file'
+        env: [{name: ANSIBLE_PARAMIKO_RECORD_HOST_KEYS}]
```

Comment:
Nit, for consistency (and arguably better readability):
```suggestion
        env: 
          - name: ANSIBLE_PARAMIKO_RECORD_HOST_KEYS
```
There's a number of similar occurrences above.